packages/@lwc/engine-dom/src/createFragment.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { isUndefined } from '@lwc/shared';
+
+const SUPPORTS_TEMPLATE = typeof HTMLTemplateElement === 'function';
+
+export let createFragment: (html: string) => Node | null;
+
+if (SUPPORTS_TEMPLATE) {
+    // Parse the fragment HTML string into DOM
+    createFragment = function (html: string) {
+        const template = document.createElement('template');
+        template.innerHTML = html;
+        return template.content.firstChild;
+    };
+} else {
+    // In browsers that don't support <template> (e.g. IE11), we need to be careful to wrap elements like
+    // <td> in the proper container elements (e.g. <tbody>), because otherwise they will be parsed as null.
+
+    // Via https://github.com/webcomponents/polyfills/blob/ee1db33/packages/template/template.js#L273-L280
+    const topLevelWrappingMap: { [key: string]: string[] } = {
+        option: ['select'],
```

Comment:
does `tfoot`, `colgroup`, `tbody` and `caption` also need to be on this map? (I don't think you can add `head`, `body` and `html` on an lwc template)

Reply:
Fixed and added a test to make sure it actually works in IE11. I did not include `head`/`body`/`html`.